numpy/core/src/multiarray/strfuncs.c

```diff
@@ -3,14 +3,25 @@
 
 #include <Python.h>
 #include <numpy/arrayobject.h>
-
 #include "npy_pycompat.h"
-
+#include "npy_import.h"
 #include "strfuncs.h"
 
 static PyObject *PyArray_StrFunction = NULL;
 static PyObject *PyArray_ReprFunction = NULL;
 
+
+static void
+npy_PyErr_SetStringChained(PyObject *type, const char *message)
```

Comment:
`_PyErr_FormatFromCause` seems to be the name used internally by python

Reply:
I think that's a better name.

Reply:
Since we use the limited API, I think we can just use `_PyErr_FormatFromCause` directly here and elsewhere.